torchreid/engine/image/am_softmax.py

```diff
@@ -172,8 +172,6 @@ def forward_backward(self, data):
                     self.scaler.update()
 
         # record losses
-        if mutual_learning:
```

Comment:
why did you remove mutual loss dumping?

Reply:
`i` variable as well as `mutual_loss` are used outside of the loop context here, which shouldn't happen. Also this value was already dumped in the loop.